tests/chunk-rollover/basic.bats

```diff
@@ -31,6 +25,7 @@ teardown_file() {
         run kubectl delete namespace "$TEST_NAMESPACE"
         rm -f ${HELM_VALUES_EXTRA_FILE}
     fi
+    unset TEST_NAMESPACE
```

Comment:
One thing this will cause though is if we have `export TEST_NAMESPACE` outside of BATS, i.e. we have done it explicitly, then this will remove it right?

Reply:
@patrick-stephens good call, I originally thought I'd need this during parallelization so that the TEST_NAMESPACE wouldn't be stuck based off the first job that ran in the parallelized testing subprocess, but it appears bats is actually creating a new subprocess per test file at the time they can run, so this can just be removed.